aliases/create_issue.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: "create_issue"
+pack: "st2-community"
+action_ref: "github.create_issue"
+formats:
+  - "issue create {{user}}/{{repo}} {{title}}"
```

Comment:
wow, was not aware that you can populate multiple parameters in an action aliases based on the message content, this would be a great example for http://docs.stackstorm.com/aliases.html